Source/Kernel/Concepts/Configuration/Observers.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Cratis. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Cratis.Chronicle.Concepts.Configuration;
+
+/// <summary>
+/// Represents the configuration for observers.
+/// </summary>
+public class Observers
+{
+    /// <summary>
+    /// Gets the timeout in seconds for observer calling its subscriber.
+    /// </summary>
+    public int SubscriberTimeoutInSeconds { get; init; } = 5;
+
+    /// <summary>
+    /// Gets the maximum number of retries that can be attempted on a failed observer partition.
+    /// </summary>
+    /// <remarks>
+    /// 0 represents infinite number of retries.
+    /// </remarks>
+    public int MaxRetryAttempts { get; init; } = 10;
+
+    /// <summary>
+    /// Gets the delay for attempting to retry a failed partition in seconds.
+    /// </summary>
+    public int RetryDelayInSeconds { get; init; } = 1;
```

Comment:
I realize after some thinking (I know, that can be dangerous), maybe it would be better to use the terminology **Backoff**, since that is well understood in resilience lingo.

e.g. `BackoffDelayInSeconds`

Another question that also popped into my head; the "InSeconds" part, is that needed, or is it sufficient just to keep it documented in the XML comment and in general documentation?

Then it would be `BackoffDelay`